lib/utility.rb

```diff
@@ -11,6 +11,7 @@
 require 'utility/elasticsearch/index/mappings'
 require 'utility/elasticsearch/index/text_analysis_settings'
 require 'utility/environment'
+require 'utility/error_monitor'
```

Comment:
this needs to be added to the connectors_utility.gemspec

While you're at it, can you add an ALL CAPS comment to this file that tells editors that any extra additions here need to be added to the gemspec (this ruined my evening last night, so it's top of mind. :) )

Reply:
OMG yes